db/migrate/20230124175942_create_users.rb

```diff
@@ -11,3 +11,4 @@ def change
   end
 end
 
+
```

Comment:
- [ ] [optional] I have looked at your migration files and some action is not well placed for instance the posts table need to be created before the comments table since you cannot reference a table that you haven't created yet, kindly consider changing the order in which the tables are created